### DISCUSSION TITLE
The waypoint legend on the handheld sonar obscures the display of the surrounding environment.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13650] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The waypoint legend on a handheld sonar takes up a significant portion of the scanning area, making it impossible to perceive the information about nearby targets.

Please add on the handheld sonar panel option "turn off" the legend in the sonar settings.
On the submarine sonar this option also will be useful.

### Reproduction steps

Select an outpost with a long name as your destination. 
Or take several tasks to destroy fauna.
You will find that the sonar has become useless since most of the scanning area is occupied by task legends.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.2.8.0 (Winter Update hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Perhaps these labels should extend outwards rather than obstruct the map
![sonar](https://github.com/FakeFishGames/Barotrauma/assets/122838333/7de9db47-9b0e-46bb-bf84-82a06b827301)

**Comment by pavel4win** [comment]
Just adding option "turn off marks" on the handheld sonar panel will be enough.
Marks on the map make it difficult to collect loot from the corpses of monsters; there is no other harm.
On this sonar screenshot there are 4 corpses hidden under the mark

![sonar](https://github.com/FakeFishGames/Barotrauma/assets/47295383/8680b3c7-f9d7-4336-b085-aa77db1712e2)
